<launch>
    <node name="rc_input_node" pkg="sunray_uav_control" type="rc_control_node" output="screen">
        <param name="uav_id" value="1" />
        <param name="uav_name" value="uav" />
        <param name="channel_arm" value="5" />
        <param name="channel_mode" value="6" />
        <param name="channel_land" value="7" />
        <param name="channel_kill" value="8" />
        <param name="value_arm" value="1" />
        <param name="value_disarm" value="-1" />
        <param name="value_mode_init" value="-1" />
        <param name="value_mode_rc" value="0" />
        <param name="value_mode_cmd" value="1" />
        <param name="value_land" value="1" />
        <param name="value_kill" value="1" />
        <param name="rc_topic" value="/uav1/sunray/fake_rc_in" />
        <!-- <param name="rc_topic" value="/uav1/mavros/rc/in" /> -->
    </node>
</launch>
